packages/minicaml/minicaml.0.2.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+version: "0.2.1"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/minicaml"
+bug-reports: "https://github.com/0x0f0f0f/minicaml/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/minicaml.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://github.com/0x0f0f0f/minicaml"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+    "dune" {>= "2.0"}
+    "ocaml"
+    "ANSITerminal"
+    "menhir"
+    "ppx_derive" {build}
```

Comment:
This package does not exist. As far as I see you depend on `ppx_deriving`. This comes with a runtime, so it is not just a `build` dependency
```suggestion
    "ppx_deriving"
```